packages/standalone/src/agent/persistent-cli-process.ts

```
@@ -0,0 +1,658 @@
+        } else if (event.is_error) {
+          const error = new Error(event.error || 'Unknown error');
+          this.currentCallbacks?.onError?.(error);
+          this.state = 'idle';
+          this.currentReject?.(error);
+          this.resetRequestState();
+        }
+        break;
+
+      case 'error':
+        this.clearRequestTimeout();
+        const error = new Error(event.error || 'Unknown error');
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)